apps/stylelint.config.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+  extends: [
+    'stylelint-config-standard',
+    'stylelint-config-standard-scss',
+    'stylelint-config-prettier-scss'
+  ],
+  rules: {
+    'no-descending-specificity': null,
+    'rule-empty-line-before': null,
+    'selector-class-pattern': null,
+    'length-zero-no-unit': null,
+    'selector-id-pattern': null,
+    'property-no-vendor-prefix': null,
+    'declaration-empty-line-before': null,
+    'scss/at-mixin-argumentless-call-parentheses': null,
```

Comment:
Why not autofix some of these more frequent violations? Looks like this one at least (picked one at random to google) can be autofixed?

Reply:
I fixed that one! It was smaller than I expected.

Some of the fixable ones, such as `'length-zero-no-unit'` still have 32 problems when I try to run the `--fix` script with it, and running the script itself results in 30 changed files. I opted to try to keep this PR relatively small, but am down to fix others if they're easy.

<img width="500" alt="image" src="https://user-images.githubusercontent.com/9142121/184676507-917baa9a-58a0-47c6-9360-8245c674bd6c.png">

Reply:
Does the linter let you auto-annotate (and allow) existing violations, but enable the rules? I worry we'll end in a place where we never get back to reenabling these rules, and will miss out on the benefits of having them. Cautionary tale is in our Ruby linting, where we have a similar list of "to dos" that didn't get touched for 4 years (although funnily enough Elijah has made some improvements recently):

https://github.com/code-dot-org/code-dot-org/blob/staging/.rubocop_todo.yml

Reply:
I don't see an auto-annotate script. I don't want this PR to get too big, but I do think we should (a) figure out our biggest priorities of rules to enable, and then (b) ensure we enable them before summer swarm is over, even if that means disabling certain files or rules.

I can start making follow-up PRs branched off of this one that tackles individual rules.

Reply:
This tackles the length-zero-no-unit rule: https://github.com/code-dot-org/code-dot-org/pull/47624

Reply:
I think you're totally right to try and get this enabled before the swarm is over, even if not perfect 👍